subsys/bluetooth/host/hci_core.c

```diff
@@ -3687,6 +3690,34 @@ const char *bt_get_name(void)
 #endif
 }
 
+uint16_t bt_get_appearance(void)
+{
+#if defined(CONFIG_BT_DEVICE_APPEARANCE_DYNAMIC)
+	return bt_dev.appearance;
+#else
+	return CONFIG_BT_DEVICE_APPEARANCE;
+#endif
+}
+
+#if defined(CONFIG_BT_DEVICE_APPEARANCE_DYNAMIC)
+int bt_set_appearance(uint16_t new_appearance)
+{
+	if (IS_ENABLED(CONFIG_BT_SETTINGS)) {
+		int err = settings_save_one("bt/appearance", &new_appearance,
+		                            sizeof(new_appearance));
```

Comment:
@jhedberg 
> You generally also seem to have trouble aligning the split lines - they should start right after the appropriate ( on the preceding line.

The style guide you gave me is not accepted by the compliance tests. How do I format this?

```
-:176: ERROR:CODE_INDENT: code indent should use tabs where possible
#176: FILE: subsys/bluetooth/host/hci_core.c:3693:
+^I^I                            sizeof(new_appearance));$
```

Reply:
@Thalley Your suggestion will not look aligned for anyone using a tab stop other than 8 characters. I intentionally used tabs for the indentation and spaces for alignment. (Do we mandate that tab stop is always 8 characters? But then what is the point of using tabs?)

Reply:
I renamed the variable to avoid this issue.

Reply:
> (Do we mandate that tab stop is always 8 characters? But then what is the point of using tabs?)

Tabs are 8 spaces as specified by https://www.kernel.org/doc/html/v4.10/process/coding-style.html#indentation (which we follow) :) I can't answer the question of "why" :D